piControlMain.c

```diff
@@ -488,11 +479,7 @@ static void __exit piControlCleanup(void)
 // false: system is not operational
 bool isRunning(void)
 {
-	if ((piDev_g.machine_type == REVPI_CORE ||
-	     piDev_g.machine_type == REVPI_CORE_SE ||
-	     piDev_g.machine_type == REVPI_CONNECT_SE ||
-	     piDev_g.machine_type == REVPI_CONNECT) &&
-	    (piCore_g.eBridgeState != piBridgeRun)) {
+	if (piDev_g.pibridge_rs485_supported == 1 && piCore_g.eBridgeState != piBridgeRun) {
```

Comment:
Per convention, booleans aren't compared to 1 in an if-clause. Just use `pibridge_rs485_supported`, which evaluates to true in an if-clause if the variable is non-zero.

Reply:
Per section 2 of the <a href="https://docs.kernel.org/process/coding-style.html#breaking-long-lines-and-strings">kernel coding style, "The preferred limit on the length of a single line is 80 columns."

So please wrap after the `&&`.